master/conf.sls

```diff
@@ -1,16 +0,0 @@
-{% from "master/map.jinja" import master, master_config with context %}
-
-include:
-  - master
-
-master-config:
-  file.managed:
-    - name: {{ master.conf_file }}
-    - source: salt://master/templates/conf.jinja
-    - template: jinja
-    - context:
-      config: {{ master_config }}
-    - watch_in:
-      - service: master
-    - require:
-      - pkg: master
```

Comment:
It was never used. It's just a leftover from the cookiecutter template.


Regards,

Tobias Macey

Senior DevOps Engineer

MIT Office of Digital Learning

http://odl.mit.edu

On Wednesday, April 20, 2016 8:56:59 AM EDT Brandon DeRosier wrote:

In master/conf.sls<https://github.com/mitodl/master-formula/pull/2#discussion_r60436469>:

> -{% from "master/map.jinja" import master, master_config with context %}

> -

> -include:

> -  - master

> -

> -master-config:

> -  file.managed:

> -    - name: {{ master.conf_file }}

> -    - source: salt://master/templates/conf.jinja

> -    - template: jinja

> -    - context:

> -      config: {{ master_config }}

> -    - watch_in:

> -      - service: master

> -    - require:

> -      - pkg: master

Why is this plan no longer necessary?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/mitodl/master-formula/pull/2/files/596af3f4a2ae3c91315ba42c3df4ec15b26e0d0e#r60436469>